app/main.go

```diff
@@ -242,6 +242,10 @@ func (app *SpacemeshApp) startSpacemesh(cmd *cobra.Command, args []string) {
 	// start p2p services
 	log.Info("Initializing P2P services")
 	swarm, err := p2p.New(Ctx, app.Config.P2P)
+
+	// Gossip messages are validated before propagating. broadcast test protocol isn't
+	api.ApproveAPIGossipMessages(Ctx, swarm) // todo: maybe only enable in tests?
```

Comment:
I agree with the TODO, we wouldn't want to allow anyone to spam the network. It should be configurable with a default to false